ows.go

```diff
@@ -39,7 +39,10 @@ import (
 var configMap map[string]*utils.Config
 
 var (
-	port = flag.Int("p", 8080, "Server listening port.")
+	port            = flag.Int("p", 8080, "Server listening port.")
+	serverDataDir   = flag.String("data_dir", utils.DataDir, "Server data directory.")
```

Comment:
Why is this needed when you can pass `--datarootdir` and `--sysconfdir` to `configure`?